SPECS/calamares/calamares.spec

```diff
@@ -1,13 +1,13 @@
 # This package depends on automagic byte compilation
 # https://fedoraproject.org/wiki/Changes/No_more_automagic_Python_bytecompilation_phase_2
-%global _python_bytecompile_extra 1
```

Comment:
Thanks for the fix. Let me know if you see any other failures related to 8266 - I'm searching through our specs now to see if anything else fell over from those changes.